OneBlink.SDK/models/Job.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using Newtonsoft.Json;
+
+namespace OneBlink.SDK.Model
+{
+    public class JobDetail
+    {
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public string key
+        {
+            get; set;
+        }
+        public string title
+        {
+            get; set;
+        }
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public string description
+        {
+            get; set;
+        }
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+        public string type
+        {
+            get; set;
+        }
+    }
+
+    public class NewJob
```

Comment:
I would make this class internal

Reply:
Can we create a constructor on this class with the required fields, i.e. `details`, `formId`, `username`